README.md

```diff
@@ -26,6 +26,7 @@ Create a `.env` file containing the following keys. Consult `.env.example` if yo
 * `SERVER_ID` (optional) - either `global` or the id of the server the bot will mainly be deployed in. Application commands will be
   registered here. If absent, application commands won't be registered.
 * `EMOJI_SERVERS` (optional) - a list of server IDs that the bot will search for emoji within.
+* `ALLOW_GIT_UPDATE` (optional) - a `true` or `false` value that determines whether the bot can automatically load data from the repository.
```

Comment:
Could this come with a health warning about being careful over enabling unless you control both the git repo and the superuser quite carefully